### ISSUE TITLE
[BUG] Fails to build under Go 1.16

### ISSUE THREAD

**Issue by vfaronov** [gotk3/gotk3#726]
I’m running Ubuntu 20.04 with GTK 3.24:
```
$ lsb_release -dcr
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

$ dpkg -l | grep libgtk-3-dev
ii  libgtk-3-dev:amd64                         3.24.20-0ubuntu1                      amd64        development files for the GTK library
```

On a fresh checkout of gotk3:
```
$ git clone https://github.com/gotk3/gotk3.git

$ cd gotk3/

$ git rev-parse HEAD
3091555944e560d3aadcf6bbe84080ebdc52a502
```

I can build the library with Go 1.15:
```
$ go version
go version go1.15.7 linux/amd64

$ go build github.com/gotk3/gotk3/gtk
```

but not with Go 1.16:
```
$ go version
go version go1.16rc1 linux/amd64

$ go build github.com/gotk3/gotk3/gtk
gtk/accel.go:272:5: val.accel_flags undefined (type _Ctype_struct__GtkAccelKey has no field or method accel_flags)
gtk/accel.go:285:22: obj.accel_flags undefined (type *_Ctype_struct__GtkAccelKey has no field or method accel_flags)
```

**Comment by hfmrow** [comment]
https://github.com/gotk3/gotk3/pull/687

**Comment by essial** [comment]
1.16 is the current release now. This project does not work with it, and the reference PR has been dropped. Any plans on supporting the current version of golang?

**Comment by hfmrow** [comment]
Work well with 1.16 (using source repository master branch, [not the actual released version 0.5.2])
I'm currently using it.

**Comment by essial** [comment]
Perfect! I'm a-ok with using the master branch. It just wasn't immediately obvious that only the master branch was compatible. Thanks so much!

**Comment by seadra** [comment]
Not using go mod either here. How can we use the master branch with go get? Importing "github.com/gotk3/gotk3" and running to go get fetches the tagged version 0.5.2 which eventually fails with error.

**Comment by hfmrow** [comment]
Hi,
Maybe doing it manually, download Zipped version and extract to your existing gotk3 local folder (respecting folder tree)
Or with git using clone repository specifying master branch
```bash
#! /usr/bin/env bash
echo "Updating Gotk3 library ?: (y/N): "
read RESP
if [ "$(echo "$RESP" |  tr "A-Z" "a-z")" = "y" ]; then
	echo "Install required 'dev' packages for Gotk3 ?: (y/N): "
	read RESP	
	if [ "$(echo "$RESP" |  tr "A-Z" "a-z")" = "y" ]; then
		sudo apt update
		sudo apt install -y libgtk-3-dev libcairo2-dev libglib2.0-dev
	fi
	
	echo "'Cleaning caches' before (B)uild or just (C)lone Gotk3 package ?: (b/C): "
	read RESP	
	if [ "$(echo "$RESP" |  tr "A-Z" "a-z")" = "b" ]; then
		echo "Delete Golang's caches ..."
		go clean -i -r -x -cache -modcache -testcache $GOPATH/src/*
		echo "Building Gotk3, may take some time ..."
		go get github.com/gotk3/gotk3/gtk
	else
		echo "Cloning Gotk3 library ..."
		echo "Remove: $GOPATH/src/github.com/gotk3/gotk3 ..."	
		rm --force --recursive $GOPATH/src/github.com/gotk3/gotk3
		echo "Processing ..."	
		git clone https://github.com/gotk3/gotk3 $GOPATH/src/github.com/gotk3/gotk3
		echo "Done ..."	
	fi
	echo "Done !!"
fi
```
This is a piece of my global golang environment update (gotk3 part)
'Y' first question, and 'enter' for the rest.
this will REMOVE your actual gotk3 directory and clone content of this repository at 'Master' in your $GOPATH/src/github.com/ directory.
Note: Rename your actual gotk3 directory before doing anything !!

**Comment by seadra** [comment]
Yes, this is manually doable, but it'd be nice if go get just worked.

As far as I understand, go get builds can be fixed either by removing all gotk3 release tags, or making a new point release.

**Comment by grandchild** [comment]
```
go get github.com/gotk3/gotk3@master

**Comment by emad-elsaid** [comment]
Getting master now doesn't work. I'm getting this error:
```
# github.com/gotk3/gotk3/glib
../../go/pkg/mod/github.com/gotk3/gotk3@v0.5.3-0.20210305115817-fdbeed35eb7d/glib/glib.go:236:3: expected '}', found 'case'
../../go/pkg/mod/github.com/gotk3/gotk3@v0.5.3-0.20210305115817-fdbeed35eb7d/glib/glib.go:269:2: expected declaration, found rv
```

**Comment by grandchild** [comment]
@emad-elsaid then pick another commit that works. Go to https://github.com/gotk3/gotk3/commits/master and start at the top picking commits one after the other by copying the short hashes on the left and use that after the `@` instead of `master`. Until you find one that works.

**Comment by seadra** [comment]
I think it would benefit the community a lot if there is a new point release that just works with go 1.16.

**Comment by emad-elsaid** [comment]
This commit 289cfb6dbf32de11dd2a392e86de4a144ac6be48  worked for me: 
```
go get -u github.com/gotk3/gotk3@289cfb6dbf32de11dd2a392e86de4a144ac6be48
```

**Comment by manen** [comment]
Shouldn't there be a new patch release that finally solves this issue without having to go bleeding-edge?

**Comment by MJacred** [comment]
@vfaronov: a release with the required fix  -> GOTK3 Version 0.6.0 has been released some time ago, please close this issue
